NAMESPACE

```diff
@@ -40,3 +40,10 @@ export(zarr_open_array)
 export(zarr_open_group)
 export(zarr_save_array)
 export(zb_slice)
+importFrom(R6,R6Class)
+importFrom(memoise,memoise)
+importFrom(memoise,timeout)
+importFrom(qs,lz4_compress_raw)
+importFrom(qs,lz4_decompress_raw)
+importFrom(qs,zstd_compress_raw)
+importFrom(qs,zstd_decompress_raw)
```

Comment:
checks complain if we import in DESCRIPTION but don't have any explicit imports from the package.

Reply:
I like the explicitness of the `::` but had not realized it was affecting the checks